docker-compose.yml

```diff
@@ -27,4 +27,3 @@ services:
 networks:
```

Comment:
`networks` is not needed, since `ESHOSTS` does not contain docker container names/addresses.

Reply:
ALSO: I think it's preferable to export production services at their native port number (news-search-ui at 8501) for minimum confusion.  Having fixed published port numbers means that production and staging can't be run on the same stack, but I think that's OK.  A good place to run the staging stack would be on the same server that runs the mcweb-staging deployment, since that would likely be the best way to test new versions of news-search-api

Reply:
> `networks` is not needed, since `ESHOSTS` does not contain docker container names/addresses.

Sure on the ESHOSTS, but the `ui` still references the `api` using the container name `APIURL: http://api:8000/v1`, so I guess retaining a custom defined network rather than using the default docker bridge network provides for better isolation

Reply:
Containers in the same stack should always be able to use container names for hostnames.